src/boot/efi/linux-aarch64.c

```diff
@@ -0,0 +1,161 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <efi.h>
+#include <efilib.h>
+#include <libfdt.h>
+
+#include "linux.h"
+#include "linux-aarch64.h"
```

Comment:
Personally, I feel like it's ok for this to go into linux.c/h and just do some #ifdef guarding. Keeps things simple...